common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -474,4 +474,13 @@ trait FeatureSwitches {
     exposeClientSide = true,
   )
 
+  val NewslettersRemoveConfirmationStep = Switch(
+    SwitchGroup.Feature,
+    "newsletters-remove-confirmation-step",
+    "Remove confirmation step when user sign up to a newsletter",
+    owners = Seq(Owner.withEmail("newsletters.dev@guardian.co.uk")),
+    safeState = Off,
+    sellByDate = never,
```

Comment:
could we add a sell by date please so we don't forget to remove this flag eventually?